src/pydap/net.py

```diff
@@ -23,7 +24,8 @@ def GET(url, application=None, session=None):
 
 
 def raise_for_status(response):
-    if response.status_code >= 400:
+    # Raise error if status is above 300:
+    if response.status_code >= 300:
```

Comment:
What is the effect of this? If the client gets a redirect, this will raise and exception, but then what handles it?